makefile

```diff
@@ -101,10 +101,10 @@ test-ci: clear clean-dist install
 bundlesize: clear clean-dist install build
 	@bundlesize
 
-validate: clear clean-dist install tsc lint stylelint test validate-build
+validate: clean-dist install tsc lint stylelint test validate-build
```

Comment:
Do we also need to take this command off of the child commands here too? `tsc` and `test`?

Reply:
why remove `tsc` and `test` ? are they run independently

Reply:
@liywjl 
I think @oliverlloyd was only asking to remove `clear` from the sub commands not to remove the sub commands themselves :)

Reply:
@oliverlloyd 
Removed from `test`, but `tsc` didn't have one as far as I can see.

Reply:
I see 👍